contracts/src/Assets.sol

```diff
@@ -60,7 +62,7 @@ library Assets {
     ) external view returns (Costs memory costs) {
         AssetsStorage.Layout storage $ = AssetsStorage.layout();
         TokenInfo storage info = $.tokenRegistry[token];
-        if (!info.isRegistered) {
+        if (!info.isRegistered && token != address(0)) {
```

Comment:
Why not just register ether as a token, so we can avoid the complexity of this check?

Reply:
Addressed in a3e5aa4bc4dcf33624c5b6f90e32a9be5279cc8f